src/scrabble.js

```diff
@@ -1,5 +1,150 @@
-function scrabble() {
-  // write code here
+function checkIncorrect(letterArray) {
```

Comment:
checkIncorrect is bad name doesn't tell me what it is going to check for / what it is goign to return, pick a better name